default/templates/default/to-spreadsheet.html

```diff
@@ -130,6 +148,56 @@
                     $('.response-fail').removeClass('hidden');
                 });
         });
+
```

Comment:
Please review this section below, there is a lot of duplicated code here. A single callback should be enough!